mslib/msui/mss_pyui.py

```diff
@@ -767,13 +769,13 @@ def show_shortcuts(self):
         dlg.setModal(True)
         dlg.exec_()
 
-    def status(self):
-        if constants.CACHED_CONFIG_FILE is None:
-            return ("Status : System Configuration")
-        else:
-            filename = constants.CACHED_CONFIG_FILE
-            head_filename, tail_filename = os.path.split(filename)
-            return("Status : User Configuration '" + tail_filename + "' loaded")
+    # def status(self):
```

Comment:
commented out sections should not appear in non-draft/WIP merge requests.

Reply:
I'll remove them.